packages/wp-now/src/wp-now.ts

```diff
@@ -158,6 +162,14 @@ export default async function startWPNow(
 		username: 'admin',
 		password: 'password',
 	});
+
+	if (
+		isFirstTimeProject &&
+		[WPNowMode.PLUGIN, WPNowMode.THEME].includes(options.mode)
```

Comment:
Should we also activate everything in `wp-content` mode?

Reply:
💥, I think it is a different use case. But it makes sense to me.
@wojtekn, Would you like to code that functionality?

Reply:
@sejas @wojtekn Let's hold off activating everything in `wp-content` mode until we receive some explicit feedback indicating such.